pkg/kv/kvserver/consistency_queue.go

```diff
@@ -143,6 +145,14 @@ func consistencyQueueShouldQueueImpl(
 		if shouldQ, priority = shouldQueueAgain(now.ToTimestamp(), lpTS, data.interval); !shouldQ {
 			return false, 0
 		}
+
+		// Bump priority by up to +1 if the replica is for a range that has been
+		// recently modified.
+		// TODO(pavelkalinnikov): Consider bumping up to +K.
+		cliff := data.lastUpdateNanos + data.interval.Nanoseconds()
```

Comment:
Doesn't this mean that we only add the priority modifier when `lastUpdateNanos` is within `interval` of the current time? We're never going to schedule the replica for a check within that interval anyway, because of `shouldQueueAgain`, so I don't think this ever takes effect?

Reply:
You're right, I think I must've somehow mixed up `lastUpdateNanos` with `QueueLastScheduled` or something.